config_robot/panda_grasp_data.yaml

```diff
@@ -82,7 +82,7 @@ hand:
     # it will also set the suction_regions_in_contact_ variable in the grasp candidates so that you are able
     # to only enable suction on regions that are in contact with the grasp target.
     # Note that this metric assumes that suction regions are equally sized
-    # ie. width = active_suction_range_x / suction_regions_x
+    # ie. width = active_suction_range_x / suction_cols_count
```

Comment:
I corrected this to the variable name that is used, but perhaps it would be better to do the reverse, replacing all instances of `suction_cols_count` with `suction_regions_x` etc?